### PR TITLE
Cinderlib: Add manual steps for installation

### PR DESCRIPTION
**Pull request by rollandf**
Changes proposed in this pull request:

- Add manual steps needed to install the cinderlib libraries

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @rollandf 

This pull request needs review by: @bennyz @shenitzky

### CODE REVIEW

File: source/develop/release-management/features/storage/cinderlib-integration.html.md [hunk 1]
```diff
@@ -267,6 +267,32 @@ Cinderlib will be packaged in RPM.
...
+$ yum-config-manager --enable openstack-pike
+$ yum install -y openstack-cinder python-pip
+$ pip install cinderlib
+```
+
+On all the hosts: (Vdsm needs to be restarted after installing the packages)
+
+```
+$ yum install -y centos-release-openstack-pike
+$ yum install -y python2-os-brick
```
Comment (nirs) [inline source/develop/release-management/features/storage/cinderlib-integration.html.md]: Don't we need to enable the repo like we do on engine?

If we have packages in yum, why we don't require them in vdsm?

File: source/develop/release-management/features/storage/cinderlib-integration.html.md [hunk 2]
```diff
@@ -285,4 +311,5 @@ OS-Brick needs to installed on the hosts
 
 ### Documentation / External references
 [1] [Cinder Library](https://github.com/Akrog/cinderlib)
+
```
Comment (nirs) [inline source/develop/release-management/features/storage/cinderlib-integration.html.md]: Looks like unrelated and unintended change

File: source/develop/release-management/features/storage/cinderlib-integration.html.md [hunk 3]
```diff
@@ -267,6 +267,32 @@ Cinderlib will be packaged in RPM.
...
+$ yum install -y centos-release-openstack-pike
+$ yum install -y python2-os-brick
+```
+
+
+For Ceph backend, the following package is needed in the Engine and in all the hosts:
+
+```
+$ yum install ceph-common
+```
```
Comment (nirs) [inline source/develop/release-management/features/storage/cinderlib-integration.html.md]: What about Fedora? maybe packaging is in better shape on Fedora?

Comment (avihaie) [reply source/develop/release-management/features/storage/cinderlib-integration.html.md]: Looks good , just 2 comments:

1) About installing 'ceph-common' on the hosts, do we need any restart (VDSM) afterwards? if so please add that restart is required.
2) Perhaps its best to change all 'install' to 'yum install -y'.

File: source/develop/release-management/features/storage/cinderlib-integration.html.md (hunk 1 above)
Comment (rollandf) [reply source/develop/release-management/features/storage/cinderlib-integration.html.md]: The package is available in yum only with the openstack repo installed.
We will need to have them available via oVirt channel.
